### PR TITLE
[master] fix: add deprecation notice for elasticsearch module/states

### PR DESCRIPTION
**Pull request by cesan3**
### What does this PR do?

Adds deprecation warning for the elasticsearch module and states to privilege the use of the extension module located in https://github.com/cesan3/salt-ext-elasticsearch.  temporarily until we set up https://github.com/salt-extensions/saltext-elasticsearch correctly to package/release the extension.

### What issues does this PR fix or reference?
Fixes: Related to https://github.com/salt-extensions/extension_migration/issues/32

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [ ] Docs
- [ ] Changelog - https://docs.saltproject.io/en/master/topics/development/changelog.html
- [ ] Tests written/updated

### Commits signed with GPG?
Yes

Please review [Salt's Contributing Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

**Review by s0undt3ch — APPROVED** [review]
So, what's really holding up the repo move to the extensions org?

**Comment by cesan3** [comment]
Thanks, @s0undt3ch . What is holding me back is that I was given a maintainer role for the organization's repo, but not enough privileges to set up the actions and secrets to release and publish the package to PyPi

**Pull request closed without merging**
